From e42680bd25b6b06b96805d8bbcf8cd7071c10a47 Mon Sep 17 00:00:00 2001
From: Ethan Chen <intervigil@gmail.com>
Date: Sun, 10 Feb 2019 12:12:50 -0800
Subject: [PATCH 26/30] kernel: Move full kernel build guard flag below all
 targets

* The Pixel DTBO generation step should also be guarded by the full
  kernel build flag.

Change-Id: I06d230b1b32ad9d974f90bfe6ae82008f4454d83
---
 build/tasks/kernel.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/build/tasks/kernel.mk b/build/tasks/kernel.mk
index 8cd2e660..9cbfb1d9 100644
--- a/build/tasks/kernel.mk
+++ b/build/tasks/kernel.mk
@@ -276,8 +276,6 @@ alldefconfig:
 	env KCONFIG_NOTIMESTAMP=true \
 		 $(call make-kernel-target,alldefconfig)
 
-endif # FULL_KERNEL_BUILD
-
 TARGET_PREBUILT_DTBO = $(PRODUCT_OUT)/dtbo/arch/$(KERNEL_ARCH)/boot/dtbo.img
 $(TARGET_PREBUILT_DTBO): $(AVBTOOL)
 	echo -e ${CL_GRN}"Building DTBO.img"${CL_RST}
@@ -289,6 +287,8 @@ $(TARGET_PREBUILT_DTBO): $(AVBTOOL)
 		--partition_name dtbo $(INTERNAL_AVB_DTBO_SIGNING_ARGS) \
 		$(BOARD_AVB_DTBO_ADD_HASH_FOOTER_ARGS)
 
+endif # FULL_KERNEL_BUILD
+
 ## Install it
 
 ifeq ($(NEEDS_KERNEL_COPY),true)
-- 
2.17.1

